.circleci/config.yml

```diff
@@ -51,15 +56,14 @@ workflows:
     jobs:
       - publish:
           name: publish
-          context: gemfury-download # Needed to build, the upload token is in the CI env vars
+          context: code-validator # Needed to build, the upload token is in the CI env vars
```

Comment:
Is the comment still true?
```suggestion
          context: code-validator
```

Reply:
yes, because it's public, we upload to pypi and the token is in the CI env vars :)

Reply:
I guess I can just remove it though 🤷